core_lib/src/graphics/vector/colourref.cpp

```diff
@@ -51,3 +53,12 @@ bool ColourRef::operator!=(ColourRef colourRef1)
         return false;
     }
 }
+
+QDebug operator<<(QDebug debug, const ColourRef& colourRef)
```

Comment:
Another minor thing, I guess the return type would be better to be `QDebug&` not `QDebug`